video2dataset/dataloader/transform.py

```diff
@@ -75,18 +75,20 @@ def _get_rand_reference(self, resize_size, h, w):
     def _get_resize_size(self, frame):
         """gets resize size"""
         orig_h, orig_w = frame.shape[:2]
+        scaling_factor = 1
         if self.resize_size is not None:
             if isinstance(self.resize_size, int):
                 f = self.resize_size / min((orig_h, orig_w))
                 resize_size = [int(round(orig_h * f)), int(round(orig_w * f))]
+                scaling_factor = 1 / f
```

Comment:
are you sure we need to do this here and just add more info instead of just computing this with the original_weight/height which is already added to additional info?

My thoughts are - this is an additional thing you are returning in the decoder that is ONLY needed for optical flow that CAN be easily computed using the info already returned by the decoder/dataloader. So I'd think to just do this outside the decoder